### PR TITLE
Ephesus: Update from `master`

### PR DESCRIPTION
**Pull request by zeeshanakram3**
- Based on #4718
- Includes QN fixes from(#4693) into Ephesus

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202078489349362/1204329553384797) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by Lezek123 — COMMENTED** [review]
Looks good, just added small comment about CLI's README

File: cli/README.md
Comment (Lezek123) [inline cli/README.md:1]: I'm seeing some small diffs after running `yarn workspace @joystream/cli oclif-dev readme` locally:
image

Comment (zeeshanakram3) [reply cli/README.md:1]: Addressed in [cb14bfa](https://github.com/Joystream/joystream/pull/4727/commits/cb14bfad17ff1954aafdb65008f75ea545f29ca4)

**Review by Lezek123 — APPROVED** [review]
LGTM

**Pull request merged**